lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -30,12 +30,12 @@ fetch = (url, options = {}) ->
   progressBar?.start()
 
   if transitionCacheEnabled and cachedPage = transitionCacheFor(url.absolute)
-    fetchHistory cachedPage
+    fetchHistory cachedPage, recallScrollPosition: false
```

Comment:
I found the recall hard to understand here. It's ambiguous as to whether or not it means remembering or redacting.